x-pack/plugins/siem/public/components/page/overview/overview_network/index.tsx

```diff
@@ -22,9 +28,30 @@ export interface OwnProps {
 export const OverviewNetwork = pure<OwnProps>(({ endDate, poll, startDate }) => (
   <EuiFlexItem>
     <EuiPanel>
-      <EuiTitle>
-        <h2>{i18n.NETWORK_HEADING}</h2>
-      </EuiTitle>
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem>
+          <EuiTitle>
+            <h2>
+              <FormattedMessage
+                id="xpack.siem.overview.networkTitle"
+                defaultMessage="Network Ingest Indices"
+              />
+            </h2>
+          </EuiTitle>
```

Comment:
If you prefer the `FormattedMessage` technique over the `i18n` technique and are migrating keys to it as you write code I would remove the `i18n.NETWORK_HEADING` if it's no longer being used from that file.